db/__init__.py

```diff
@@ -78,3 +78,18 @@ class TransactionStatusMemo(Base):
     id = Column(Integer, primary_key=True)
     memo_body = Column(String, nullable=False)
     tx_signatures_id = Column(Integer, ForeignKey(f'{SCHEMA}.tx_signatures.id'), nullable=False)
+
+
+class AmmLiquidity(Base):
+    __tablename__ = 'amm_liquidity'
+
+    id = Column(Integer, primary_key=True)
```

Comment:
the key is composed of other collumns, so i think this is not needed (don't quote me on that, but CLOBLiqudity has similar key and doesn't need `id` here)

Reply:
:+1: